app/src/main/java/com/udacity/nanodegree/blooddonation/util/PermissionUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.udacity.nanodegree.blooddonation.util;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.pm.PackageManager;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+
+/**
+ * Created by tavishjain on 20-04-2018.
+ */
+
+public class PermissionUtils{
+
+    static Activity mActivity;
+
+    public PermissionUtils(Activity activity){
+        mActivity = activity;
+    }
+
+    static int REQUEST_CODE_LOCATION_AND_SMS = 1;
```

Comment:
REQUEST_CODE_LOCATION_AND_SMS defined at multiple places, put it in a constant file

Reply:
or make it public
 public static int REQUEST_CODE_LOCATION_AND_SMS = 1;